Objects/longobject.c

```diff
@@ -132,6 +132,15 @@ long_normalize(PyLongObject *v)
     return v;
 }
 
+#if PyLong_MAXFREELIST > 0
+static struct _Py_long_state *
```

Comment:
Given how little is done here, might be worth declaring `inline` to ensure no function call overhead paid for such cheap work.